channels/chan_usbradio.c

```diff
@@ -4463,7 +4493,7 @@ static void tune_write(struct chan_usbradio_pvt *o)
 		CONFIG_UPDATE_SIGNAL(txmixa, txmixa, mixer_type);
 		CONFIG_UPDATE_SIGNAL(txmixb, txmixb, mixer_type);
 		if (ast_config_text_file_save2(CONFIG, cfg, "chan_usbradio", 0)) {
-			ast_log(LOG_WARNING, "Failed to save config\n");
+			ast_log(LOG_WARNING, "Failed to save config %s.\n", CONFIG);
```

Comment:
Personally, I don't think periods are needed at the end of log messages, it doesn't help readability any, though I don't know if that's a hard guideline, so this is more of a suggestion if you agree or you can ignore if not

Reply:
Poof!